src/form.js

```diff
@@ -179,7 +179,7 @@ define([
                 form.hashtagMap = _.clone(vellum.datasources.getHashtagMap({}));
                 form.invertedHashtagMap = _.invert(form.hashtagMap);
                 form.hashtagTransformations = vellum.datasources.getHashtagTransforms({});
-                form.hasCaseHashtags = true;
+                form.hasCaseHashtags = vellum.datasources.isReady();
```

Comment:
Yes, that's what happens now (see [line 140](https://github.com/dimagi/Vellum/blob/542a4e27dabf10b715645a64601a965df240208f/src/form.js#L140)) when rich text is enabled. If rich text is disabled then it won't be flipped until rich text is enabled.